src/explorer.ts

```diff
@@ -348,28 +371,17 @@ class KubernetesNamespaceResource extends KubernetesResource {
     }
 }
 
-class KubernetesServiceFolder extends KubernetesResourceFolder {
-    constructor() {
-        super(kuberesources.allKinds.service);
+class KubernetesPodSelectorFolder extends KubernetesResourceFolder {
+    constructor(readonly kind: kuberesources.ResourceKind, readonly fn: (kubectl: Kubectl) => Promise<kubectlUtils.PodSelector[]>) {
+        super(kind);
     }
 
     async getChildren(kubectl: Kubectl, host: Host): Promise<KubernetesObject[]> {
-        const services = await kubectlUtils.getServices(kubectl);
+        const services = await this.fn(kubectl);
```

Comment:
Is `services` still the correct name here?

Reply:
fixed.